source/guide/serialization.md

```
@@ -407,6 +407,60 @@ by a C++ `grove<genomic_coordinate, std::string>`; with labelled edges the inter
 `grove<genomic_coordinate, std::string, std::string>`. Typed `BedGrove` / `GffGrove` `.gg` files
 round-trip the structured `BedEntry` / `GffEntry` payloads.
 
+### Partial random-access reading
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)